website/docs/quick-start-guide.md

```diff
@@ -220,6 +220,13 @@ CREATE TABLE hudi_table (
 PARTITIONED BY (city);
 ```
 
+:::note NOTE:
```

Comment:
This is nice callout. Thanks for thinking about this. But in the spirit of keeping the quickstart guide short and crisp, here is my recommendation. 

- We should replace the note with a link to the same note in the SQL DDL page on any issues.  
- Also, clarify the quickstart Setup instructions to explicitly state this quickstart assumes you have default setup and database settings for spark-hive integration.